src/pages/ProductList/ProductList.js

```diff
@@ -12,6 +12,45 @@ import './ProductList.scss';
 
 const ProductList = () => {
   const [productData, setProductData] = useState([]);
+  const [searchParams, setSearchParams] = useSearchParams();
+  const location = useLocation();
+  const url = location.search;
+
+  useEffect(() => {
+    fetch(`http://10.58.52.223:3000/products/list${url}`)
```

Comment:
변수명이 url에서 search로 바뀌었으니 여기도 
```suggestion
    fetch(`http://10.58.52.223:3000/products/list${search}`)
```

위처럼 바꾸면 좋을 것 같습니다

Reply:
@YunKukPark 님, 말씀하신 불필요한 변수 수정 완료하여 pr 날렸습니다! 
확인해주시면 머지하겠습니다 감사합니다 :)